include/simdjson/generic/ondemand/value_iterator-inl.h

```diff
@@ -504,6 +509,10 @@ simdjson_really_inline bool value_iterator::is_at_start() const noexcept {
 simdjson_really_inline bool value_iterator::is_at_container_start() const noexcept {
   return _json_iter->token.index == _start_position + 1;
 }
+simdjson_really_inline bool value_iterator::is_at_iterator_start() const noexcept {
+  auto delta = _json_iter->token.index - _start_position;
+  return delta == 1 || delta == 2;
```

Comment:
A comment here might be helpful to improve code legibility.

Reply:
:+1: